apps/hashdotdev/src/_pages/blog/0014_structured-data-value.mdx

```diff
@@ -0,0 +1,129 @@
+
+While large models learn well from unstructured data, they still learn better from structured data, and in particular high-quality, curated sets of data. It's one of the reasons that high-quality corpuses (such as that of Stack Overflow) are weighted so highly in the training texts that large language models use. And as the internet becomes increasingly full of convincing-sounding machine-generated content that can be hard or impossible to distinguish from stuff written by actual human experts, it stands to reason that training new foundation models will become more difficult in the future. While machine-generated synthetic data has many benefits, its presence in [too large a quantity](https://arxiv.org/abs/2307.01850) can degrade the quality (precision) and diversity (recall) of generative models trained using them. [HASH](https://hash.ai/) ensures that the provenance of information in a user or organization's web (their graph of data) is clear, and generated outputs are (internally at least) marked as such.
+
+Business software, and software in general, both currently operate atop structured data. Your ERP, accounting software, CRM, CMS, knowledge-base and communications software all use them. The tools you use for _unstructured_ note-taking rely on structured databases and schemas to store that information under the hood. Literally all consumer and business software is built atop information structures and structured data. That's just how software works. And absolutely all of these apps and services are today connected by a smörgåsbord of highly structured protocols that underpin operating systems as we know them, and the very internet itself.
+
+Explicitly created and defined structures will continue to have relevance in a world of powerful foundation models which are capable of inferring structure on demand.
+
+Pre-structured data will:
+
+- enable applications to interface with each other without requiring foundation models to perform a 'reconciliation' step, transforming one's outputs to another's inputs
+- provide safety and confidence through verification and validation: even where reconciliation may be conducted by foundation models, having example structured data will enable model-produced translations to be tested
+- provide a cost benefit: data does not need to be structured at the point of need, it already exists
+- provide a time benefit: data is instantly available, again not requiring computation on the fly
+- act as an always-available fallback in the event of foundation model unavailibilty or disconnection
+
+These features of pre-structured data, combined with foundation models, will unlock and mainstream new technologies and best practices.
+
+## Increasing value
+
+With thanks to both foundation models _and_ pre-structured data, we may finally see certain frontiers unlocked. One thing we're betting on at HASH is that **simulation** will finally come of age, as a widely-used tool for everyday decisionmaking.
```

Comment:
maybe?
```suggestion
With thanks to both foundation models _and_ pre-structured data, we may finally see certain frontiers unlocked. One thing we're betting on at HASH is that **simulation** will finally come of age, as a widely-used tool for informing everyday decisionmaking.
```

Reply:
Going to pass on this one. Informing will be the primary use, but hopefully substituting in certain cases as well (automatic optimization under safe parameters and bounds, etc.)